doc/examples/applications/plot_pixel_graphs.py

```diff
@@ -35,7 +35,7 @@
 _, ax = plt.subplots()
```

Comment:
```suggestion
fig, ax = plt.subplots()
```

Reply:
@lagru isn't this unnecessary, if `fig` is never used afterwards? :thinking:

Reply:
It is. I think I thought it might be slightly more confusing to break up the standard `fig, ax = plt.subplots()`. But your are right to call me out. Requesting to revert this is also noise. :sweat_smile: